<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>-31</x>
      <y>-11</y>
      <width>211</width>
      <height>481</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50, 50, 50);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="registrar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font-size: 14px;
    border-style: none;	
	color: white;
}

QPushButton::hover {
    border-style: solid;	
	background-color: rgb(110, 110, 110);
}</string>
     </property>
     <property name="text">
      <string>Registrar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font-size: 14px;
    border-style: none;	
	color: white;
}

QPushButton::hover {
    border-style: solid;	
	background-color: rgb(110, 110, 110);
}

</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>420</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>@docafavarato</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="esqueci">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	font-size: 14px;
    border-style: none;	
	color: white;
}

QPushButton::hover {
    border-style: solid;	
	background-color: rgb(110, 110, 110);
}

</string>
     </property>
     <property name="text">
      <string>Esqueci meus dados</string>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="Pages">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>-1</y>
      <width>601</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(104, 104, 104);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 20px;</string>
      </property>
      <property name="text">
       <string>Usuário:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 20px;</string>
      </property>
      <property name="text">
       <string>Senha:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 20px;</string>
      </property>
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>200</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 20px;</string>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="usuarioIn">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
background-color: rgb(216, 216, 216);
border-style: none;</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="senhaIn">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
background-color: rgb(216, 216, 216);
border-style: none;</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomeIn">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
background-color: rgb(216, 216, 216);
border-style: none;</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="enviar">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>270</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: white;
	border-style: none;
}

QPushButton::pressed {
	background-color: rgb(110, 110, 110);
}</string>
      </property>
      <property name="text">
       <string>Enviar</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="emailIn">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>210</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
background-color: rgb(216, 216, 216);
border-style: none;</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 20px;</string>
      </property>
      <property name="text">
       <string>Usuário:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="usuarioLo">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
background-color: rgb(216, 216, 216);
border-style: none;</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 20px;</string>
      </property>
      <property name="text">
       <string>Senha:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="senhaLo">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
background-color: rgb(216, 216, 216);
border-style: none;</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhHiddenText</set>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>32767</number>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="logar">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>160</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: white;
	border-style: none;
}

QPushButton::pressed {
	background-color: rgb(110, 110, 110);
}</string>
      </property>
      <property name="text">
       <string>Logar</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 20px;</string>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="emailEs">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white;
background-color: rgb(216, 216, 216);
border-style: none;</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="enviarEmail">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>110</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: white;
	border-style: none;
}

QPushButton::pressed {
	background-color: rgb(110, 110, 110);
}</string>
      </property>
      <property name="text">
       <string>Enviar</string>
      </property>
      <property name="shortcut">
       <string>Return</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>220</y>
        <width>461</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 16pt &quot;Segoe UI&quot;;
color: white;</string>
      </property>
      <property name="text">
       <string>Você receberá um email com instruções
para recuperar os seus dados. Lembre-se
de anotá-los para não esquecer mais.</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>773</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>registrar</tabstop>
  <tabstop>login</tabstop>
  <tabstop>esqueci</tabstop>
  <tabstop>usuarioIn</tabstop>
  <tabstop>senhaIn</tabstop>
  <tabstop>nomeIn</tabstop>
  <tabstop>emailIn</tabstop>
  <tabstop>enviar</tabstop>
  <tabstop>usuarioLo</tabstop>
  <tabstop>senhaLo</tabstop>
  <tabstop>logar</tabstop>
  <tabstop>emailEs</tabstop>
  <tabstop>enviarEmail</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
